app/assets/stylesheets/common/base/group.scss

```diff
@@ -77,6 +78,18 @@ span.mention-group {
       width: $icon-size;
     }
   }
+
+  .group-details-button {
+    display: flex;
+    flex-shrink: 0;
+    gap: 10px;
```

Comment:
does it work on safari ?

Reply:
Yup, the screenshots were taken in Safari 😃